Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -41,14 +41,15 @@ public void PreProc_EndToEnd_SuccessCase()
             // Checks end-to-end happy path for the pre-processor i.e.
             // * arguments are parsed
             // * targets are installed
+            // * build wrapper installer is called
```

Comment:
Minimal test coverage at the moment: just checks that the build wrapper installer is called in the happy-path case.